meshtastic/tests/test_smoke1.py

```
@@ -1,118 +1,182 @@
 import pytest
 
 from ..util import findPorts
 from .cli_test_utils import _quote_shell_path, run_cli_with_timeout
 
-# seconds to pause after running a meshtastic command
 PAUSE_AFTER_COMMAND = 2
 PAUSE_AFTER_REBOOT = 7
 PAUSE_AFTER_REBOOT_COMMAND = 18
+RESTORE_ATTEMPTS = 8
+RESTORE_RETRY_DELAY_SECONDS = 10
+DEFAULT_URL_TOKEN = "CgUYAyIBAQ"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "DEFAULT_URL_TOKEN"

(S105)